src/main/resources/springModelMapperAdvice.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you rename the file so it will have the same naming convention as the other files in this folder? E.g. spring-modelmapper-advice.xml?

Reply:
Done